acceptance/acceptance_test.go

```diff
@@ -988,3 +995,85 @@ func prepareWheelBuildDirectory(t *testing.T, dir string) string {
 
 	return latestWheel
 }
+
+func applyBundleConfig(t *testing.T, tmpDir string, bundleConfig map[string]any) string {
```

Comment:
Can you include a docstring saying what this function returns?

Reply:
Done. Also changed it to return bool. https://github.com/databricks/cli/pull/2809/commits/3a991f453a325279f66ff525e782a661de194998